### DISCUSSION TITLE
About Accounts

### DISCUSSION THREAD

**Discussion by MrFanservice-png** [anidl/multi-downloader-nx#709] [Q&A]
Do you need a premium account from Crunchyroll to download premium videos, or will a free account will do just fine?

If that's not possible, how do I only download the subtitles from Crunchyroll?

**Comment by Jaynator495** [comment]
You need a premium account to download premium videos, but a free account will be able to download free videos. As for subtitles, while I think it may be possible, I don't think the tool allows even selecting episodes you don't have access to

**Reply by MrFanservice-png** [reply]
So it's not possible to download subtitles from premium videos with a free account?
